README.md

```diff
@@ -25,14 +25,14 @@ flowchart LR
 
 ### Operator ###
 
-Určuje URL adresy ke stažení pro _downloader_. Za přidávání URL ke zpracování je zodpovědný pavouk. Nové URL pavouk získává z naparsované podoby dat, kterou vytváří _parser_.
+The Operator determines URL addresses for the Downloader. A Spider is responsible for adding the URLs, which it gets from the parsed form of the data provided by the Parser.
 
-Činnost _operatoru_ se skládá z:
+The Operator’s job is to:
 
-1. inicializace pavouka,
-2. kontroly existence seznamu nových URL,
-3. načítání nových URL ze strukturovaných dat,
-4. zařazování těchto URL do fronty pro _downloader_.
+1. initialize a Spider,
+2. check for the existence of a new URL list,
```

Comment:
```suggestion
2. check for the existence of a new URLs,
```

Reply:
The original version mentions URL lists (“seznam nových URL“), as in files with the .urls extension. That’s why I kept it in the translation.
The suggestion is grammatically incorrect. A plural “URLs” does not conform to the usage of the indefinite article “a.”

```suggestion
2. check for the existence of new URLs,
```

Why the change from the “URL lists” to “URLs”? I don’t want to merge before we resolve this dispute.

Reply:
to not be so much specific but yes if we translate czech version then it should be keep it, deletion should be done in czech if so.

Reply:
I moreover think that we should delete the whole bullet point. **The Downloader** checks the existence of the files with URLs, also called URL lists.


“
1. Initialize a spider,
2. extract new URLs from structured data
”
looks more precise to me.